DEVELOP.md

```diff
@@ -9,65 +9,66 @@ Dependencies:
+### 1. Install Dependencies
+
+```bash
+brew install Python                #brew will install the latest Python 2.7.9, which includes pip
+brew install postgres				
+brew install rabbitmq				
+pip install virtualenv
+```
+
+
+### 2. Set up CycleDash
 
 ```bash
 virtualenv venv                    # Initialize a new virtual environment.
-source venv/bin/activate           # Activate your virtual environment.
+source venv/bin/activate           # Activate your virtual environment. This is the only command you will need to run every time you start the app.
 pip install -r requirements.txt    # Install python requirements into virtualenv
-npm install                        # Install JS requirements
+npm install                        # Install JS requirements and packages in package.json
 make initenv                       # Initialize environment file.
-$EDITOR ENV.sh                     # Fill in values.
+$EDITOR ENV.sh                     # Fill in values. (see next step)
 ```
 
 
-### Database
+### 3. Edit the ENV.sh file (generated by `make initenv`)
 
-We use PostgreSQL as our datastore, with the schema described in `schema.sql`.
-
-You can load the schema with `psql cycledash < schema.sql`, and you'll be good
-to go.
-
-On OS X, setting up and running psql might look like this:
+Environment variables which must be exported--edit them in the ENV.sh file make
+made for you.
 
 ```
-brew install postgres
-postgres -D /usr/local/var/postgres
-createdb cycledash
-psql cycledash
-psql cycledash < schema.sql
+export PORT=5000
+export DATABASE_URI=postgres:///cycledash'
+export CELERY_BACKEND='db+sqlite:///celery.db'
+export CELERY_BROKER='amqp://localhost'
+export WEBHDFS_USER=username
+export WEBHDFS_URL=http://example.com:5000
+export TYPEKIT_URL="yourtypekitURLwithfontsincluded"
 ```
 
 
-### Start CycleDash
-
-To start the application server:
+### 4. Set up the Database (Postgres)
 
-```bash
-gulp prod
-./run.sh
-```
+We use PostgreSQL as our datastore, with the schema described in `schema.sql`.
 
-Start a worker to process the queue. Make sure RabbitMQ is up and running!
+On OS X, setting up and running psql might look like this:
 
+```bash
+postgres -D /usr/local/var/postgres
+createdb cycledash
+psql cycledash
+psql cycledash < schema.sql        # This loads the schema
 ```
-./worker.sh Bob # Or whatever you want to name your worker.
-                # Say, RosieTheRiveter.
-```
-
-You can start more workers with `./worker.sh <name>` etc. *with a different
-name*.
 
 
-### JavaScript
+### 5. Set up the JavaScript
 
 You can iterate more quickly on JavaScript with the following:
 
 ```
-npm install             # Installs all packages in package.json.
 npm install gulp -g     # Make sure you have gulp installed.
-gulp                    # Compile the JS and start the automatic compiler
-                        # and live-reloader.
+gulp                    # Compile the JS and start the automatic compiler and live-reloader.
```

Comment:
I'd keep this on two lines; under the 80 char archaic line length limit we try to stick to.